### ISSUE TITLE
Sub-waypoints

### ISSUE THREAD

**Issue by waynegraham** [scholarslab/NeatlineWaypoints#2]
We got a feature request for "sub-waypoints" today in a show-and-tell demo. Right now the plugin can order the elements, but they would like to have a feature to express a parent-child relationship. From a front-end perspective, I think if we added the ability to pull waypoints left and right we could accomplish this (ignoring the actual implementation details of the tree structure).

**Comment by davidmcclure** [comment]
Perhaps something like this: http://mjsarfatti.com/sandbox/nestedSortable/?

**Comment by waynegraham** [comment]
This is exactly the kind of thing I was thinking. The nice thing about this is that it has serialize methods that may make this relatively simple to implement...

**Comment by davidmcclure** [comment]
Yep, saving the data wouldn't be that different from saving the ordering, which we do already. We'd ned to add some kind of `parent` field to the waypoint model, though.
